server/datastore/mysql/migrations/tables/20250211141712_AddHostCertificatesTable.go

```diff
@@ -0,0 +1,44 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250211141712, Down_20250211141712)
+}
+
+func Up_20250211141712(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE host_certificates (
+	cert_id                     BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+	host_id                     INT UNSIGNED NOT NULL,
+	not_valid_afte              DATETIME(6) NOT NULL,
+	not_valid_before            DATETIME(6) NOT NULL,
+	certificate_authority 		TINYINT(1) NOT NULL,
+	common_name                 VARCHAR(255) NOT NULL,
+	key_algorithm               VARCHAR(255) NOT NULL,
+	key_strength                INT NOT NULL,
+	key_usag                    VARCHAR(255) NOT NULL,
+	serial                      VARCHAR(255) NOT NULL,
+	signing_algorithm           VARCHAR(255) NOT NULL,
+	subject_country             VARCHAR(2) NOT NULL,
+	subject_org                 VARCHAR(255) NOT NULL,
+	subject_org_unit            VARCHAR(255) NOT NULL,
+	subject_common_name         VARCHAR(255) NOT NULL,
+	issuer_country              VARCHAR(2) NOT NULL,
+	issuer_org                  VARCHAR(255) NOT NULL,
+	issuer_org_unit             VARCHAR(255) NOT NULL,
+	issuer_common_name          VARCHAR(255) NOT NULL,
+	checksum                    BINARY(20) NOT NULL,
```

Comment:
This is the `sha1` field in the osquery payload? I feel like we should reflect the hashing algorithm in the name, as it's likely we will move to sha256 or something else in the future (that is, I think osquery will probably provide something else, sha1 being [proven "broken"](https://crypto.stackexchange.com/a/3691), and we will have to follow suit, ideally in a distinct column to avoid ambiguity).

Reply:
Or store the hashing algorithm as a prefix, at the expense of slightly larger storage (and textual instead of binary), but that requires some processing before using the value (e.g. to validate the cert's bytes) and it's not obvious that there is a prefix in there.